src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -186,6 +191,28 @@ struct MazeSetTrackAction : public GameActionBase<GAME_COMMAND_SET_MAZE_TRACK, G
         return std::make_unique<GameActionResult>();
     }
 
+    GameActionResult::Ptr QueryOther(GameActionResult::Ptr res) const
+    {
+        TileElement* tileElement = map_get_surface_element_at(_x / 32, _y / 32);
+        if (tileElement == nullptr)
+        {
+            res->Error = GA_ERROR::UNKNOWN;
+            res->ErrorMessage = STR_INVALID_SELECTION_OF_OBJECTS;
+            return res;
+        }
```

Comment:
This looks quite odd to me. The error message is weird, but why does it even bother to fetch the surface element to begin with? It doesn't do anything with it.

Reply:
This is part of the query. It is checking so that it doesn't crash as part of the execute. This is fine IMHO.